### PR TITLE
Jjdzr4 mw 21 formularz dodawania nowego produktu

### PR DESCRIPTION
**Pull request by MateuszLesisz**

### CODE REVIEW

File: myFit-webApp/src/main/resources/messages.properties [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+addProductPage.title=New Product
+addProductPage.headLine=Wype?nij formularz aby doda? produkt do bazy danych!
```
Comment (miosz) [inline myFit-webApp/src/main/resources/messages.properties:3]: bad file encoding?

File: myFit-webApp/src/main/resources/templates/product-new.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <meta charset="UTF-8">
+    <title th:text="#{addProductPage.title}"></title>
+    <meta charset="UTF-8">
+</head>
+<body>
+<div style="text-align: center">
+    <h1 th:text="#{addProductPage.headLine}"></h1>
+    <form th:object="${product}" th:action="@{/user/product/new}" th:method="post">
+
+        <label th:for="name_input"> Name </label><br>
```
Comment (miosz) [inline myFit-webApp/src/main/resources/templates/product-new.html:13]: what about validations?

File: myFit-webApp/src/main/resources/templates/product-new.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        <input type="text" id="name_input" th:field="*{name}"><br>
+
+        <label th:for="kcalPer100g_input">Kcal w 100g</label><br>
```
Comment (mateuszostrowski) [inline myFit-webApp/src/main/resources/templates/product-new.html:16]: we can use th:text and invoke to messages.properties
html: label th:for="kcalPer100g_input" th:text="#{100g.kcal}"></label>
messages.properties: 100g.kcal = Kcal w 100g

File: myFit-webApp/src/main/resources/templates/product-new.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        <input type="number" id="kcalPer100g_input" th:field="*{kcalPer100g}"><br>
+
+        <label th:for="fatPer100g_input">Tłuszcz w 100g</label><br>
```
Comment (lyingparachute) [inline myFit-webApp/src/main/resources/templates/product-new.html:19]: messages.properties: 100g.fat = Tłuszcz w 100g

File: myFit-webApp/src/main/resources/templates/product-new.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        <input type="number" id="fatPer100g_input" th:field="*{fatPer100g}"><br>
+
+        <label th:for="carbohydratesPer100g_input">Węglowodany w 100g</label><br>
```
Comment (lyingparachute) [inline myFit-webApp/src/main/resources/templates/product-new.html:22]: Tak jak wyżej

File: myFit-webApp/src/main/resources/templates/product-new.html [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        <input type="number" id="carbohydratesPer100g_input" th:field="*{carbohydratesPer100g}"><br>
+
+        <label th:for="proteinPer100g">Białko w 100g</label><br>
```
Comment (lyingparachute) [inline myFit-webApp/src/main/resources/templates/product-new.html:25]: Tak jak wyżej

File: myFit-webApp/src/main/resources/templates/product-new.html [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+        <input type="number" id="proteinPer100g" th:field="*{proteinPer100g}"><br>
+        <br>
+        <button type="submit">Dodaj</button>
```
Comment (lyingparachute) [inline myFit-webApp/src/main/resources/templates/product-new.html:28]: messages.properties: add = Dodaj
th:text="#{addl}"

**Review by lyingparachute — APPROVED** [review]

File: myFit-webApp/src/main/resources/messages.properties (hunk 1 above)
Comment (MateuszLesisz) [reply myFit-webApp/src/main/resources/messages.properties:3]: Done

File: myFit-webApp/src/main/resources/templates/product-new.html (hunk 2 above)
Comment (MateuszLesisz) [reply myFit-webApp/src/main/resources/templates/product-new.html:13]: Validation will be add when we start adding product to database

File: myFit-webApp/src/main/resources/templates/product-new.html (hunk 3 above)
Comment (MateuszLesisz) [reply myFit-webApp/src/main/resources/templates/product-new.html:16]: done

File: myFit-webApp/src/main/resources/templates/product-new.html (hunk 4 above)
Comment (MateuszLesisz) [reply myFit-webApp/src/main/resources/templates/product-new.html:19]: done

File: myFit-webApp/src/main/resources/templates/product-new.html (hunk 5 above)
Comment (MateuszLesisz) [reply myFit-webApp/src/main/resources/templates/product-new.html:22]: done

File: myFit-webApp/src/main/resources/templates/product-new.html (hunk 6 above)
Comment (MateuszLesisz) [reply myFit-webApp/src/main/resources/templates/product-new.html:25]: done

File: myFit-webApp/src/main/resources/templates/product-new.html (hunk 7 above)
Comment (MateuszLesisz) [reply myFit-webApp/src/main/resources/templates/product-new.html:28]: done

**Review by AdSten — APPROVED** [review]

**Pull request merged**